prow/tide/history/history.go

```diff
@@ -45,19 +46,24 @@ type History struct {
 	sync.Mutex
 	logSizeLimit int
 
-	opener io.Opener
+	opener iov2.Opener
```

Comment:
Nit: If we define the interface here, we should use that rather than `iov2.Opener`

Reply:
Oh that was an oversight. Changed it in a few places but missed that one